### DISCUSSION TITLE
Multilingual Refiner with boolean field

### DISCUSSION THREAD

**Discussion by christouuu** [microsoft-search/pnp-modern-search#1811] [Q&A]
**Is your feature request related to a problem? Please describe.**
The need would be to have a refiner mapped to a boolean field 
Today we need to map a boolean field only to a RefinableString
Thus, depending on the crawled property that is mapped, we have "True/False" or "1/0" as value in the RefinableString value
In a multilingual SharePoint communication site, thoses values are not OK for end users. 

**Describe the solution you'd like**
Ability to declare a refiner with a RefinableString and have it act as a boolean so that it displays "Yes/No" in English, "Oui/Non" in French etc.

**Describe alternatives you've considered**
Alternative would have been to use the 0/1 value and have a custom layout to handle the display but I don't see how it can be possible to detect language and have different labels according to that
Another alternative is to use taxonomy with 2 values in the termset to make it act like a boolean. Not great great. 

**Additional context**
We use these awesome webparts in a multilingual SharePoint communication site (French, English).

**Comment by wobba** [comment]
I think we could probably pass in some context information to the filter web part, and then you could use a custom template. @FranckyC, how hard would it be to expose the SPFx context object for the filters?

**Reply by christouuu** [reply]
Would be great. Ideally, would be good to have either the page language (multilingual communication site) or the interface language
